libs/shared/util-node-env/src/lib/shared-util-node-env.ts

```diff
@@ -33,4 +33,9 @@ export const NODE_ENV = process.env.NODE_ENV,
   GTM = process.env.NX_GTM,
   INSTATUS_APP_ID = process.env.NX_INSTATUS_APP_ID,
   ALGOLIA_APP_ID = process.env.NX_ALGOLIA_APP_ID,
-  ALGOLIA_API_KEY = process.env.NX_ALGOLIA_API_KEY
+  ALGOLIA_API_KEY = process.env.NX_ALGOLIA_API_KEY,
+  QOVERY_FORUM_URL = 'https://discuss.qovery.com',
```

Comment:
You should create `NX_*`  env variable in the github repo settings. Otherwise if it should remain const values, it should be migrated in another util library.

Reply:
I created a library `shared/util-const` but we have already an `shared/enums` library with some constants, let me know what you prefer